suprb/optimizer/rule/es/es.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 
 from suprb.rule import Rule, RuleInit
-from suprb.rule.matching import MatchingFunction, OrderedBound
```

Comment:
Is centre correct or is it center?

Reply:
Probably Center since Wilson is American, changed it.